README.md

```diff
@@ -1,24 +1,31 @@
-[![Join the chat at https://gitter.im/zlsa/atc](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zlsa/atc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
There's an awesome one that shows how many people are on it, and on click takes you to the link of your choosing. Part of slackin I think... I'll find it later